src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,92 @@
-/* eslint-disable */
-import React from 'react';
+import { useDispatch } from 'react-redux';
```

Comment:
Consider using `useAppDispatch` if you have a custom hook for dispatching actions. This can help ensure type safety when dispatching actions in a TypeScript environment.